addons/hw_drivers/led_manager.py

```diff
@@ -0,0 +1,67 @@
+import platform
+import subprocess
+import time
+from threading import Thread, Timer
+from odoo.addons.hw_drivers.tools import helpers, wifi
+
+STATUS_UPDATE_DELAY_SECONDS = 10.0
+BLINK_DELAY_SECONDS = 1.0
+
+class LedManager(Thread):
+    def run(self):
+        if platform.system() != 'Linux':
+            return
+
+        self.is_raspberry_pi_5 = helpers.get_raspberry_pi_model() == 5
+        self.green_led_on = False
+        self.red_led_on = False
+        self.blinking = False
+        self._disable_led_toggles()
+        while True:
+            if wifi.is_access_point() or not helpers.get_ip():
+                # RED if there is no internet
+                self._set_green_led(False)
+                self._set_red_led(True)
+                self.blinking = False
+            elif helpers.get_odoo_server_url():
+                # GREEN if there is internet and connected to DB
+                self._set_green_led(True)
+                self._set_red_led(False)
+                self.blinking = False
+            else:
+                # BLINKING GREEN there is internet but not yet paired
+                self._set_red_led(False)
+                if not self.blinking:
+                    self.blinking = True
+                    self._blink_green_led()
+            time.sleep(STATUS_UPDATE_DELAY_SECONDS)
+
+    def _disable_led_toggles(self):
+        subprocess.run("echo none | sudo tee /sys/class/leds/ACT/trigger", shell=True)
+        subprocess.run("echo none | sudo tee /sys/class/leds/PWR/trigger", shell=True)
+
+    def _set_green_led(self, value):
+        if self.green_led_on == bool(value):
+            return
+        self.green_led_on = bool(value)
+        if self.is_raspberry_pi_5:
+            # 'ACT' LED is inverted on Raspberry Pi 5
+            value = not value
+        subprocess.run(f"echo {'1' if value else '0'} | sudo tee /sys/class/leds/ACT/brightness", shell=True)
+
+    def _set_red_led(self, value):
+        if self.red_led_on == bool(value):
+            return
+        self.red_led_on = bool(value)
```

Comment:
```suggestion
        if self.red_led_on is value:
            return
        self.red_led_on = value
```
do we really need to cast?